pkg/bgpv1/agent/controller.go

```diff
@@ -188,17 +174,30 @@ func (c *Controller) Run(ctx context.Context) {
 		})
 	)
 
-	// add an initial signal to kick things off
-	c.Sig.Event(struct{}{})
-
 	l.Info("Cilium BGP Control Plane Controller now running...")
+	ciliumNodeCh := c.CiliumNodeResource.Events(ctx)
 	for {
 		select {
+		case ev, ok := <-ciliumNodeCh:
+			if !ok {
+				l.Info("LocalCiliumNode resource channel closed")
```

Comment:
I would add something like "..., Cilium BGP Control Plane Controller shut down" to the log, so that the consequence is logged as well